src/decisionengine_modules/GCE/transforms/GceFigureOfMerit.py

```diff
@@ -41,8 +41,15 @@ def transform(self, data_block):
             it = row["InstanceType"]
             entry_name = row["EntryName"]
 
-            occupancy_df = gce_occupancy[((gce_occupancy.AvailabilityZone == az) & (gce_occupancy.InstanceType == it))]
-            occupancy = float(occupancy_df["Occupancy"].values[0]) if not occupancy_df.empty else 0
+            try:
+                occupancy_df = gce_occupancy[
+                    ((gce_occupancy.AvailabilityZone == az) & (gce_occupancy.InstanceType == it))
+                ]
+            except:
```

Comment:
Do you know which exceptions can be risen? Is it possible to catch specific exceptions?

Reply:
This is taken care of.